manual/manual/cmds/ocamldoc.etex

```diff
@@ -494,8 +494,8 @@ type weather2 =
 
 (** The comment for type my_record *)
 type my_record = {
-    val foo : int ;    (** Comment for field foo *)
-    val bar : string ; (** Comment for field bar *)
```

Comment:
This was grossly invalid. Is it possible that using one of the `{caml_example}` environments would have caught the mistake earlier? If applicable in this case, I would recommend using them as part of this PR -- if we fix, let's fix long-term. cc @Octachron who may be able to comment on applicability.

Reply:
In this case, the code example is a signature; this would require a small tweak to `caml_tex2.ml` to add an option to wrap code example inside a module type declaration.